go.mod

```diff
@@ -4,7 +4,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1 // indirect
 	github.com/Masterminds/squirrel v0.0.0-20190107164353-fa735ea14f09
 	github.com/VividCortex/mysqlerr v0.0.0-20170204212430-6c6b55f8796f
-	github.com/argoproj/argo v2.2.0+incompatible
+	github.com/argoproj/argo v2.3.0-rc2+incompatible
```

Comment:
Missed that before.
Same thing as [this comment](https://github.com/kubeflow/pipelines/pull/1289/files#r282031123)